src/lib/pickles/tests/test_ro.ml

```diff
@@ -0,0 +1,20 @@
+(* initially written to see the consistency between updates of digestif *)
+let test_bits_random_oracle_consistency_check () =
+  let s = "BitsRandomOracle" in
+  let exp_output_str =
+    "01000001101110001111111100011000011000100010100011010010110110010011101101101000001110001110100101010100001000001110101110111010"
+  in
+  let exp_output =
+    List.map (fun i -> i = '1') (List.of_seq (String.to_seq exp_output_str))
+  in
+  let output = Pickles.Ro.bits_random_oracle ~length:128 s in
```

Comment:
We don't really need to export `Ro` as part of the interface for Pickles to test it. We could use `Pickles__Ro` as well. 

I'm not sure this is per se a good practice to do that, nor is it probably to expose a module of a library that is, as far as it looks, purely local.

Was it necessary to update `alcotest` to run this test?

Reply:
No, it was not necessary to update alcotest for this test.
It is part of testing the update to digestif, in addition to adding a small test.

I agree with your points regarding exposong `Ro`. I'm not in favor of using `Pickles__Ro`, and I am more used to expose the module to test it. Fine doing it if you want.

There is a way to bypass it by exposing a module `Pickles_internal` to unit test functions if we don't want to fully exposed Ro in the API.

Another way is to use inline tests. However, I remember we were thinking about moving to alcotest.

Reply:
Well. I prefer moving to `alcotests`, mostly for engineering reasons. Shall we make an issue of exposing `Pickles_internal` then?